doc/langref.html.in

```diff
@@ -777,6 +777,30 @@
       implementation feature, not a language semantic, so it is not guaranteed to be observable to code.
       </p>
       {#header_close#}
+
+      {#header_open|Destructuring#}
+      <p>
+        Some aggregate values ({#link|Tuples#}, {#link|Arrays#}, {#link|Vectors#}) can be destructed using
```

Comment:
"destructed" should be "destructured".  Or rephrase the sentence to avoid so many destructure*, maybe:  A destructuring assignment can separate elements of an aggregate value ({#link|Tuples#}, {#link|Arrays#}, {#link|Vectors#}):

Reply:
Reworded.